### PR TITLE
feature: complete e2e ticker selection

### PR DESCRIPTION
**Pull request by iprado78**

### CODE REVIEW

File: components/TickerSearch/TickerSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+function OptionsList({ options }: { options: TickerOption[] }) {
+  return (
+    <ul className={styles.dropdown}>
+      {options.map((option) => (
+        <li key={option.value}>
+          <button type="button" className={styles.dropdown__option} onClick={
+            () => {
+              onTickerSelection(option.value)
+            }
+            }>
```
Comment (gracescharf) [inline components/TickerSearch/TickerSearch.tsx:65]:
```suggestion
          }>
```

File: components/TickerSearch/TickerSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      ))}
+    </ul>
+  )
+}
+
+export default function TickerSearch() {
+  const options = useOptions()
+
+  return (
+    <form className={styles.search}>
```
Comment (gracescharf) [inline components/TickerSearch/TickerSearch.tsx:78]: some functionality options:
1. the user is able to submit the form and send the input value as the stock ticker payload
2. the user is not able to submit the form, and must choose from the buttons in the drop down (so would need to re-add the preventDefault
3. the user is able to submit the form only if the input value matches one of the options
4. the user is able to submit the form and the first option is sent as the stock ticker payload

**Review by gracescharf — APPROVED** [review]
So nice :)

File: components/TickerSearch/TickerSearch.tsx (hunk 2 above)
Comment (iprado78) [reply components/TickerSearch/TickerSearch.tsx:78]: Created new issue with suggestions

**Pull request merged**